firejail/virtualbox.profile

```diff
@@ -0,0 +1,22 @@
+# virtualbox profile
```

Comment:
We don't even have virtualbox on the shell-server. Why do we need to customize config for a non-existed software?